broadinstitute/single_cell_portal_core
# fixing bug in multi-gene study search where study has no clusters

To test:

0. run `SyntheticStudyPopulator('cow_blood')` to populate a study with expression data but no clusters
1. do a global gene search for `agpat2` and `apoe`
2. Confirm the Bovine blood study appears, with the text saying "This study contains agpat2, apoe.  This study does not have cluster data to support visualization in the portal". (as opposed to a red error message
